newIDE/app/src/MainFrame/index.js

```diff
@@ -2160,6 +2160,11 @@ const MainFrame = (props: Props) => {
                   projectItemName: editorTab.projectItemName,
                   setPreviewedLayout,
                   onOpenExternalEvents: openExternalEvents,
+                  onOpenEvents: (sceneName: string) =>
+                    openLayout(sceneName, {
+                      openEventsEditor: true,
+                      openSceneEditor: false,
```

Comment:
just to confirm, this is not closing the scene editor right?

Reply:
Correct, this is telling to open the events editor and to not open the scene editor if it's not open already. If it's opened already (which is most probably the case), it stays untouched. In particular, it's not closed :)